webview/src/setup/components/dvc/ProjectUninitialized.tsx

```diff
@@ -1,29 +1,24 @@
 import React, { PropsWithChildren } from 'react'
+import { useSelector } from 'react-redux'
 import { GitUninitialized } from './GitUnitialized'
 import { DvcUninitialized } from './DvcUnitialized'
+import { SetupState } from '../../store'
 
 interface ProjectUninitializedProps {
-  canGitInitialize: boolean | undefined
   initializeDvc: () => void
   initializeGit: () => void
-  needsGitInitialized: boolean | undefined
 }
 
 export const ProjectUninitialized: React.FC<
   PropsWithChildren<ProjectUninitializedProps>
-> = ({
-  initializeDvc,
-  needsGitInitialized,
-  canGitInitialize,
-  initializeGit,
-  children
-}) => {
+> = ({ initializeDvc, initializeGit, children }) => {
```

Comment:
Same here (should probably review exported functions from `messages.ts` in general).

Reply:
Fixed and reviewed the functions!